Tests/SentryTests/Integrations/Performance/UIViewController/SentryTimeToDisplayTrackerTest.swift

```diff
@@ -0,0 +1,168 @@
+import Foundation
+import Sentry
+import XCTest
+
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+
+class SentryTimeToDisplayTrackerTest: XCTestCase {
+
+    private class Fixture {
+        let dateProvider: TestCurrentDateProvider = TestCurrentDateProvider()
+        let tracer = SentryTracer(transactionContext: TransactionContext(operation: "Test Operation"), hub: nil)
+        let spanCreation: SpanCreationCallback = { op, desc in
+            return SentrySpan(context: SpanContext(trace: SentryId(), spanId: SpanId(), parentId: nil, operation: op, spanDescription: desc, sampled: .yes))
+        }
+    }
+
+    private let fixture = Fixture()
+
+    override func setUp() {
+        super.setUp()
+        CurrentDate.setCurrentDateProvider(fixture.dateProvider)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        clearTestState()
+    }
+
+    func testreportInitialDisplay_notWaitingFullDisplay() {
+        let sut = SentryTimeToDisplayTracker(for: UIViewController(), waitForFullDisplay: false)
+        let tracer = fixture.tracer
+
+        fixture.dateProvider.setDate(date: Date(timeIntervalSince1970: 7))
+        tracer.addExtension(sut)
+        XCTAssertEqual(tracer.children.count, 1)
+
+        let ttidSpan = tracer.children.first
```

Comment:
Using `XCTUnwrap` would specifically help us know if there wasn't a span like we were expecting, and avoid all the optional unwrapping/coalescing downstream:
```suggestion
        let ttidSpan = try XCTUnwrap(tracer.children.first, "Expected a TTID span")
```